Makefile

```diff
@@ -51,12 +53,14 @@ else
 endif
 
 export TARGET_OUT ?= $(shell pwd)/out/$(TARGET_OS)_$(TARGET_ARCH)
+export TARGET_OUT_LINUX ?= $(shell pwd)/out/linux_amd64
 
 ifeq ($(BUILD_WITH_CONTAINER),1)
 export TARGET_OUT = /work/out/$(TARGET_OS)_$(TARGET_ARCH)
+export TARGET_OUT_LINUX = /work/out/linux_amd64
 CONTAINER_CLI ?= docker
 DOCKER_SOCKET_MOUNT ?= -v /var/run/docker.sock:/var/run/docker.sock
-IMG ?= gcr.io/istio-testing/build-tools:master-2019-11-08T17-21-18
+IMG ?= gcr.io/istio-testing/build-tools:master-2019-12-15T16-17-48
```

Comment:
I just noticed - this should be proxy-build-tools to match CI.

The common-files update tooling needs some work to handle this repo. I'd suggest making a change. I won't be able to fix the update logic for this repository prior to the holidays, but perhaps someone else will be able to.

Reply:
@clarketm FYI ^

Reply:
Thanks for noticing this! I added a script to check tag of proxy build tools. The script will be run as part of presubmit lint check.